e2e/deployers/subscription.go

```diff
@@ -0,0 +1,85 @@
+package deployers
+
+import (
+	"github.com/ramendr/ramen/e2e/util"
+	"github.com/ramendr/ramen/e2e/workloads"
+)
+
+type Subscription struct {
+	NamePrefix string
+	McsbName   string
+
+	ChannelName      string
+	ChannelNamespace string
+}
+
+func (s *Subscription) Init() {
+	s.NamePrefix = "sub-"
```

Comment:
(note) As informed by @par97 this helps when the same workload needs to run in parallel with different deployers.

In the future we potentially would add a resource suffix that is either randomly generated or a hash of the full test name, to handle cases where we want to run the "same" combination of deployer+workload for various reasons.

Reply:
thanks, I added your comment as a comment in Subscription's NamePrefix field.